spec/models/organization_status_spec.rb

```diff
@@ -33,7 +33,10 @@
     let(:exercise2) { create :exercise }
     let(:discussion) { build :discussion, initiator: user }
 
-    before { create :assignment, submitter: user, organization: organization, exercise: exercise1 }
+    before {
+      create :assignment, submitter: user, organization: organization, exercise: exercise1
+      organization.switch!
+    }
```

Comment:
All these tests were using the default `test_organization` instead of the one created specifically for the occasion, because it was never set as current.